packages/web-components/src/switch/switch.styles.ts

```diff
@@ -0,0 +1,170 @@
+    color: ${colorNeutralForegroundDisabled};
+  }
+  .switch {
+    position: relative;
+    box-sizing: border-box;
+    width: 40px;
+    height: 20px;
+    background-color: ${colorTransparentBackground};
+    border: 1px solid ${colorNeutralStrokeAccessible};
+    border-radius: 10px;
+    outline: none;
+    cursor: pointer;
+    margin: ${spacingVerticalS} ${spacingHorizontalS};
+  }
+  .switch * {
+    transition-duration: ${durationNormal};
+    transition-timing-function: ${curveEasyEase};
+    transition-property: transform;
+  }
+  .switch:hover {
+    background: none;
+    border-color: ${colorNeutralStrokeAccessibleHover};
+  }
+  .switch:active {
+    border-color: ${colorNeutralStrokeAccessiblePressed};
+  }
+  :host([disabled]) .switch,
+  :host([readonly]) .switch {
+    border: 1px solid ${colorNeutralStrokeDisabled};
+    background-color: none;
+    pointer: default;
+  }
+  .checked-indicator {
+    position: absolute;
+    height: 14px;
+    width: 14px;
+    left: 3px;
+    top: 50%;
+    margin-top: -7px;
+    border-radius: 50%;
+    background-color: ${colorNeutralForeground3};
+  }
+  :host([aria-checked='true']) .checked-indicator {
+    background-color: ${colorNeutralForegroundInverted};
+    transform: translateX(19px);
+  }
+  :host([aria-checked='true']) .switch:hover .checked-indicator {
+    background: ${colorNeutralForegroundInvertedHover};
+  }
+  :host([aria-checked='true']) .switch:active .checked-indicator {
+    background: ${colorNeutralForegroundInvertedPressed};
+  }
+  .switch:hover .checked-indicator {
+    background-color: ${colorNeutralForeground3Hover};
+  }
+  .switch:active .checked-indicator {
+    background-color: ${colorNeutralForeground3Pressed};
+  }
+  :host([disabled]) .checked-indicator,
+  :host([readonly]) .checked-indicator {
+    background: ${colorNeutralForegroundDisabled};
+  }
+  :host([disabled]) .status-message,
+  :host([readonly]) .status-message {
+    cursor: pointer;
+  }
+  .label__hidden {
+    display: none;
+    visibility: hidden;
+  }
+  :host([aria-checked='true']) .switch {
+    background: ${colorCompoundBrandBackground};
+  }
+  :host([aria-checked='true']) .switch:hover {
+    background: ${colorCompoundBrandBackgroundHover};
+    border-color: ${colorCompoundBrandBackgroundHover};
+  }
+  :host([aria-checked='true']) .switch:active {
+    background: ${colorCompoundBrandBackgroundPressed};
+    border-color: ${colorCompoundBrandBackgroundPressed};
+  }
+  :host([aria-checked='true'][disabled]) .switch {
+    background: ${colorNeutralBackgroundDisabled};
+  }
+  :host([aria-checked='true'][disabled]) .switch .checked-indicator {
+    background: ${colorNeutralForegroundDisabled};
+  }
+  .unchecked-message {
+    display: block;
+  }
+  .checked-message {
+    display: none;
+  }
+  :host([aria-checked='true']) .unchecked-message {
+    display: none;
+  }
+  :host([aria-checked='true']) .checked-message {
+    display: block;
+  }
```

Comment:
I think when my PR goes in for updating the versions and you rebase you'll want to remove these. These won't exist at that time in the template.